src/Main.dfy

```diff
@@ -134,14 +131,39 @@ module Main {
     return Success(());
   }
 
-  method PrintExceedingStddevs(description: string, stddevs: seq<(string, real)>, limit: real) returns (passed: bool) {
-      var exceedingStddevs := Filter((t:(string, real)) => limit < t.1, stddevs);
-      if 0 < |exceedingStddevs| {
+  method PrintStatistics(options: Options, groupedResults: map<string, seq<TestResult.TestResult>>, initialPassed: bool)
+    returns (passed: bool)
+  {
+    passed := initialPassed;
+    if options.maxDurationStddev.Some? {
+      var stddevs := ResultGroupStatistics(groupedResults, results => TestResultDurationStatistics(results).stddev);
+      passed := PrintExceedingValues("duration standard deviation", stddevs, options.maxDurationStddev.value);
+    }
+
+    if options.maxDurationCV.Some? {
+      var CVs := ResultGroupStatistics(groupedResults, results => TestResultDurationStatistics(results).CV());
+      passed := PrintExceedingValues("duration CV", CVs, options.maxDurationCV.value);
```

Comment:
```suggestion
      passed := PrintExceedingValues("duration coefficient of variance", CVs, options.maxDurationCV.value);
```

When it doesn't occur in option names I think we can afford the more readable expansion.